packages/neo-one-node-http-rpc/src/setupRPCServer.ts

```diff
@@ -2,6 +2,7 @@ import { cors, setupServer } from '@neo-one/http';
 import { context, onError as appOnError } from '@neo-one/http-context';
 import { createChild, nodeLogger, rpcLogger } from '@neo-one/logger';
 import { Blockchain, Node } from '@neo-one/node-core';
+import { NativeContainer } from '@neo-one/node-native';
 import { Disposable, Labels } from '@neo-one/utils';
 import * as nodeHttp from 'http';
 import Application from 'koa';
```

Comment:
I don't think you need NativeContainer in the RPC server, blockchain already has a reference to native